Makefile

```diff
@@ -79,7 +77,7 @@ test: unit functional
 check: depend fmt vet lint
 
 unit: depend
-	cd $(DEST) && go test -tags=unit $(shell glide novendor) $(TESTARGS)
+	cd $(DEST) && go test -tags=unit $(shell go list ./... | grep -v vendor) $(TESTARGS)
```

Comment:
go list ./... Seems to have some trouble here and outputs some errors; not sure it's harmful for the unit test.  But I don't think you need to add the ignore here anyway, tested in my version without any special considerations and it matched the number of unit tests executed as the original glide version of the code.